gent_base.theme

```diff
@@ -716,3 +716,37 @@ function gent_base_preprocess_pager(&$variables) {
   $variables['items']['first']['text'] = '1';
   $variables['items']['last']['text'] = $total;
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function gent_base_preprocess_node__news__teaser(&$variables) {
+  _gent_base_generate_placeholder_image($variables);
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function gent_base_preprocess_node__event__teaser(&$variables) {
+  _gent_base_generate_placeholder_image($variables);
+}
+
+/**
+ * Helper function to get generate a placeholder teaser.
+ *
+ * This requires the system_stream_wrapper module!
+ */
+function _gent_base_generate_placeholder_image(&$variables) {
```

Comment:
Je geeft beter enkel het specifiek field door aan deze helper functie ipv het volledige node object. Zo is het ook herbruikbaar voor andere image fields.

Reply:
👍